docs/source/tutorials/psij.ipynb

```diff
@@ -0,0 +1,547 @@
+    "- Cobalt\n",
+    "- Flux\n",
+    "- LSF\n",
+    "- PBS\n",
+    "- Slurm\n",
+    "\n",
+    "Generally, when jobs are submitted, they will be queued for a variable period of time, depending on how busy the target machine is. Once the Job is started, its executable is launched and runs to completion.\n",
+    "\n",
+    "In PSI/J, a job is submitted by [JobExecutor.submit(Job)](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) which permanently binds the Job to that executor and submits it to the underlying resource manager."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Basic Usage\n",
+    "Creating a *single* job with PSI/J:\n",
+    "1. Create a [JobExecutor](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor) instance.\n",
+    "2. Create a [JobSpec](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_spec.JobSpec) object and populate it with information about your job\n",
+    "3. Create a [Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job) with that [JobSpec](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_spec.JobSpec)\n",
+    "4. Submit the [Job](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job.Job) instance to the [JobExecutor](https://exaworks.org/psij-python/#docs/.generated/psij.html/#psij.job_executor.JobExecutor)\n",
+    "\n",
+    "### Viewing Job Status\n",
+    "To view a jobs' status, you can use `print(str(Job.status))`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from psij import Job, JobExecutor, JobSpec\n",
+    "\n",
+    "environment = \"local\"\n",
+    "executable = \"/bin/date\"\n",
+    "ex = JobExecutor.get_instance(environment)\n",
+    "job = Job(JobSpec(executable=executable))\n",
+    "\n",
+    "print(\"Job Status Stages:\")\n",
+    "print(\"\\tPrior to submitting: \", str(job.status))\n",
+    "ex.submit(job)\n",
+    "print(\"\\tAfter submitting: \", str(job.status))\n",
+    "time.sleep(1)\n",
+    "print(\"\\tOnce completed: \", str(job.status))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multiple Jobs\n",
+    "This section will show you how to submit *multiple* jobs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from psij import Job, JobExecutor, JobSpec\n",
+    "from psij import JobState\n",
+    "\n",
+    "executable = \"/bin/date\"\n",
+    "ex = JobExecutor.get_instance(environment)\n",
+    "\n",
+    "jobs = []\n",
+    "\n",
+    "print(\"Submitting jobs: \")\n",
+    "for _ in range(10):\n",
+    "    job = Job(JobSpec(executable=executable))\n",
+    "    print(str(job.status))\n",
```

Comment:
is it necessary to say `str(job.status)` there or would `job.status` suffice?

Reply:
This comment has been addressed